css/style.css

```diff
@@ -16,4 +8,135 @@ body {
  * - When using Flexbox, remember the items you want to move around need to be inside a parent container set to 'display: flex'
  */
 
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+
+.logo-icon {
+  width: 30px;
+}
+
+.menu-nav {
+  list-style-type: none;
+  padding: 17px;
+  margin: 0px;
+  text-align: center;
+  font-size: 16px;
+}
+
+.nav-flex-box {
+  display: flex;
+  justify-content: center;
+  max-width: 100%;
+}
+
+.nav-list {
+  padding: 10px;
+  padding-left: 10px;
+  padding-right: 1px;
+  width: 7%;
+}
+
+.logo-icon {
+  margin-right: 200px;
+}
+
+.web-content {
+  overflow-x: hidden;
+  overflow-y: scroll;
+  height: 100%;
+}
+
+.background-img {
+  width: 100%;
+  position: absolute;
+}
```

Comment:
Instead of using an `img` tag for the large picture, try setting it as a `background-image` instead.

Reply:
It shouldn't need absolute positioning. It's better to let the page flow naturally.